exercises/gigasecond/canonical-data.json

```diff
@@ -62,6 +62,23 @@
             "moment": "2015-01-24T23:59:59"
           },
           "expected": "2046-10-03T01:46:39"
+        },
+        {
+          "uuid": "fcec307c-7529-49ab-b0fe-20309197618a",
+          "description": "does not mutate the input",
+          "comments": [
+            "This test case is for languages with a mutable date or datetime  ",
+            "datastructure. After running add(moment), moment should remain   ",
+            "the same."
+          ],
+          "scenarios": [
+            "immutable"
+          ],
+          "property": "moment",
```

Comment:
As the property is now no longer the method to be called, you could consider using a descriptive property like the one used in the [diffie-hellman exercise](https://github.com/exercism/problem-specifications/blob/master/exercises/diffie-hellman/canonical-data.json#L21). That said, the current property does work too, so feel free to ignore.